docs/modules/ROOT/pages/servlet/appendix/namespace/http.adoc

```diff
@@ -1798,6 +1798,11 @@ Specifies the period in seconds for which the remember-me cookie should be valid
 By default it will be valid for 14 days.
 
 
+[[nsa-remember-me-hashing-algorithm]]
+* **hashing-algorithm**
```

Comment:
Likewise, would recommend that this be left out for now in favor of applications using `services-ref`.